docs/gallery/howto/autogen/graph_builder.py

```diff
@@ -108,15 +146,15 @@ def add_multiply(x, y, z):
 
 
 # %%
-# Create the a Task from the workgraph (Experimental)
-# ---------------------------------------------------
+# Create a Task from the workgraph (Experimental)
+# -----------------------------------------------
```

Comment:
Since you mentioned this part at the beginning, so better to move this section before `graph_builder`.

Reply:
Is now integrated into the first part